packages/suite/src/components/suite/DropZone/index.tsx

```diff
@@ -139,7 +160,7 @@ export const DropZone = (props: Props) => {
         <Wrapper {...getWrapperProps()}>
             <StyledInput {...getInputProps()} />
             <Label>
-                <StyledIcon icon="CSV" />
+                <StyledIcon icon={props.icon || "BINARY"} />
```

Comment:
```suggestion
                {props.icon && <StyledIcon icon={props.icon} />}
```

Reply:
My intention was to use binary file icon as a default one when there is no other icon specified. Do you prefer not to display any icon in that case?

Reply:
True. Binary file icon looks like a universal file icon, so it is fine 👍